1config-ui/src/com/brunobonacci/oneconfig/ui/controller.cljs

```diff
@@ -264,11 +312,23 @@
     (.setReadOnly ace-instance editable?)
     (.setHighlightActiveLine ace-instance true)))
 
+(defn compare-ace-code-block!
+  [left right]
+  (let [
+        left (js-obj "content" left)
+        right (js-obj "content" right)
+        props (js-obj "element" "#acediff" "left" left "right" right)
+        ]
+    (js/AceDiff. props)))
 
 (defn highlight-code-block
   [editable? type]
   (js/setTimeout #(highlight-ace-code-block! editable? (get ace-theme-mapping type "json")) 75))
 
+(defn compare-code-block
+  [entries]
+  (js/setTimeout #(compare-ace-code-block! (first entries) (last entries)) 75))
```

Comment:
here we could desctructure entries into `[{:keys [left right]}]` and the just use the same to pass it to `compare-ace-code-block!`

Reply:
sure, will do that thanks